Applications/WebPortal/WebApp/library/source/components/checkmark.tsx

```diff
@@ -0,0 +1,117 @@
+import { css, StyleSheet } from 'aphrodite';
+import * as React from 'react';
+import { DisplaySize } from '../display_size';
+
+interface Properties {
+
+  /** The size to display the component at. */
+  displaySize?: DisplaySize;
+
+  /** Determines if the box is checked. */
+  isChecked: boolean;
+
+  /** Determines if the component can be edited. */
+  readonly?: boolean;
+
+  /** Called when the check mark is clicked on.
+   * @param value - The new value.
+   */
+  onClick?: (value?: boolean) => void;
```

Comment:
I don't think the ```value``` here is needed or a good idea since this component is stateless.